### PR TITLE
add implementation for flow_statistics

### PR DESCRIPTION
**Pull request by FlyingQianMM**
At each frame, count the following data from the the beginning to current frame:
* the number of identifiers 
* the number of identifiers across the horizontal center line in a top to down manner
* the number of identifiers across the horizontal center line in a down to top manner

At a manual-set interval (unit: seconds), count the number of identifiers from the the beginning to end frame.

### CODE REVIEW

File: deploy/pptracking/include/postprocess.h [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #include <opencv2/highgui/highgui.hpp>
 
 #include "include/utils.h"
+#include "include/config_parser.h"
```
Comment (jerrywgz) [inline deploy/pptracking/include/postprocess.h]: 这里不需要增加吧

File: deploy/pptracking/src/postprocess.cc [hunk 2]
```diff
@@ -98,10 +98,76 @@ cv::Mat VisualizeTrackResult(const cv::Mat& img,
...
-  throw "Not Implement";
+                   std::vector<int>* out_count_list,
+                   std::map<int, std::vector<float>>* prev_center) {
+  if (frame_id == 0) {
+    interval_count_set->clear();
+  }
+  
+  // Count in and out number
+  // Use horizontal center line as the entrance just for simplification
+  float entrance_y = entrance.top;
```
Comment (jerrywgz) [inline deploy/pptracking/src/postprocess.cc]: 这里说明下in out是怎么计算的吧

File: deploy/pptracking/src/postprocess.cc [hunk 3]
```diff
@@ -98,10 +98,76 @@ cv::Mat VisualizeTrackResult(const cv::Mat& img,
...
+  }
+  
+  fprintf(fp, "Frame id: %d, Total count: %d, In count: %d, Out count: %d",
+    frame_id, count_set->size(), in_count_list->size(), out_count_list->size());
+  // Note: if the input video duration is small than the interval.
+  // printf will not be done.
+  if (curr_interval_count > -1)
+    fprintf(fp, ", Count during %d secs: %d", secs_interval, curr_interval_count);
+  fprintf(fp, "\n");
+  fclose(fp);
```
Comment (jerrywgz) [inline deploy/pptracking/src/postprocess.cc]: 这部分逻辑放到pipeline下吧，首先在SaveMOTResult中将结果保存在records中，然后pipeline下再进行保存

File: deploy/pptracking/src/pipeline.cc [hunk 4]
```diff
@@ -236,7 +248,10 @@ void Pipeline::RunMOTStream(const cv::Mat img, const int frame_id, cv::Mat out_i
   if (count_) {
     // Count total number 
     // Count in & out number
-    PaddleDetection::FlowStatistic(result, frame_id, &count_list, &in_count_list, &out_count_list);
+    PaddleDetection::FlowStatistic(
```
Comment (jerrywgz) [inline deploy/pptracking/src/pipeline.cc]: 这里默认统计总人数吧，in & out 逻辑用输入参数控制是否进行

File: deploy/pptracking/include/postprocess.h (hunk 1 above)
Comment (FlyingQianMM) [reply deploy/pptracking/include/postprocess.h]: 已删除。Thanks.

File: deploy/pptracking/src/postprocess.cc (hunk 2 above)
Comment (FlyingQianMM) [reply deploy/pptracking/src/postprocess.cc]: 已添加注释说明。Thanks.

File: deploy/pptracking/src/postprocess.cc (hunk 3 above)
Comment (FlyingQianMM) [reply deploy/pptracking/src/postprocess.cc]: 已挪至pipeline.cpp下进行文件保存。 Thanks.

File: deploy/pptracking/src/pipeline.cc (hunk 4 above)
Comment (FlyingQianMM) [reply deploy/pptracking/src/pipeline.cc]: 已经用count来判断in & out 逻辑。 Thanks.

**Pull request closed without merging**